stdlib/Artifacts/src/Artifacts.jl

```diff
@@ -542,7 +542,21 @@ function jointail(dir, tail)
     end
 end
 
+const _artifact_str_world_age = Ref{UInt}(typemax(UInt))
```

Comment:
Should we re-use `Base._require_world_age`?

Reply:
Might as well, yeah